src/cli/configure/index.ts

```diff
@@ -118,8 +121,8 @@ export const configure = async () => {
       log.warn(log.bold('✗ Failed to install dependencies. Resume with:'));
 
       log.newline();
-      log.plain(log.bold(packageManager, 'install'));
-      log.plain(log.bold(packageManager, 'run', 'format'));
+      log.plain(log.bold(packageManager.install));
+      log.plain(log.bold(packageManager.exec, 'format'));
```

Comment:
Is this right? IIRC, `pnpm exec` is for `.bin`s and `pnpm run` is for `package.json` scripts.

Alternatively maybe we can simplify to just `pnpm` if that's reliable.